mne_bids/tests/test_write.py

```diff
@@ -288,8 +288,8 @@ def test_ctf():
     with pytest.raises(ValueError, match="not found"):
         read_raw_bids(bids_basename + '_meg.ds', output_path)
 
-    raw = read_raw_bids(bids_basename + '_meg.ds', output_path,
-                        return_events=False)
+    raw, __, __ = read_raw_bids(bids_basename + '_meg.ds', output_path,
```

Comment:
why double underscore?

Reply:
I read at some point that this is preferred, due to the special meaning  of a single underscore in some contexts ... however I couldn't find the source anymore and in fact, found more sources using a single underscore ...

so I'll do that from now on `_`